models/profile.ts

```diff
@@ -0,0 +1,15 @@
+import { model, Schema } from 'mongoose';
+import { IProfile } from '../types/profile';
+
+// A Profile for a User
+const profileSchema: Schema = new Schema<IProfile>(
+  {
+    user_id: { type: Schema.Types.ObjectId, required: true },
+    username: String,
+    bio: String,
+    photo: String,
```

Comment:
Just thinking on how we would process the binary image data (since they wouldn't necessarily have uri strings right?) and found that we could store them as `Buffer` data types, which is just binary data.

An example: https://medium.com/@alvenw/how-to-store-images-to-mongodb-with-node-js-fb3905c37e6d